src/App.js

```diff
@@ -32,9 +35,14 @@ export default {
 									contract: this.$store.state.contractAddress,
 								};
 								if (rawProof[4]) {
```

Comment:
Since there is no else clause only proofs with images will get commited to the store

Reply:
fixed